dockerfiles/redis-azure-blobs/index.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import jsonErrorHandler from 'express-json-error-handler';
+import dynamico from '@dynamico/express-middleware';
+import { AzureBlobStorage } from '@dynamico/azure-blob-storage';
+import { RedisIndexStorage } from '@dynamico/redis-index-storage';
+import { CompositionStorage } from '@dynamico/composition-storage';
+import { ContainerURL, StorageURL, AnonymousCredential } from '@azure/storage-blob';
+import redis from 'redis';
+import cors from 'cors';
+import nconf from 'nconf';
```

Comment:
y not using dotenv?

Reply:
because dotEnv is used to load a file into environment variables while what I want to achieve here is loading every configuration style (file, env or args) to a configuration object. With this you can either define a file or environment variables or arguments and get the params values... does this make sense? Am I missing something basic here?